config/checkstyle.xml

```diff
@@ -97,6 +97,15 @@
     <module name="MultipleVariableDeclarations"/>
     <module name="DefaultComesLast"/>
     <module name="EmptyStatement"/>
+    <module name="CovariantEquals" />
+    <module name="EqualsHashCode" />
+    <module name="IllegalInstantiation" />
+    <module name="StringLiteralEquality" />
+    <module name="MissingOverride" />
+    <module name="PackageAnnotation" />
```

Comment:
Since we don't use javadocs, also remove the PackageAnnotation please. It makes no sense. **Sorry, it was my fault**. The description snippet *...The JLS does highly recommend...* just convinced me not to do any further research. :fearful:

I'll add a comment to the jira ticket.